inspirehep/modules/search/query_factory.py

```diff
@@ -75,7 +75,7 @@ def invenio_query(pattern, search):
             ))
         finally:
             if current_app.debug:
-                current_app.logger.info(json.dumps(query.to_dict(), indent=4))
+                current_app.logger.debug(json.dumps(query.to_dict(), indent=4))
```

Comment:
Not relevant for this PR, but please read this writeup by @mihaibivol on how to do proper logging: https://github.com/inspirehep/inspire-next/issues/1371.